src/components/TodoItem.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/TodoItem.module.css';
+
+const TodoItem = ({
+  itemProp, setTodos, delTodo, setUpdate,
+}) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleChange = (id) => {
+    setTodos((prevState) => prevState.map((todo) => {
+      if (todo.id === id) {
+        return {
+          ...todo,
+          completed: !todo.completed,
+        };
+      }
+      return todo;
+    }));
+  };
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
+
+  const viewMode = {};
+  const editMode = {};
+  if (editing) {
+    viewMode.display = 'none';
+  } else {
+    editMode.display = 'none';
+  }
+
```

Comment:
You have done well writing this code for the edit function. 👏 👏
 However, I noticed a bug whenever the user makes changes in the input field. 

After editing the task, the user should be able to save the new task by pressing 'enter' for instance. 

Kindly make changes to fix this bug. 

![partner review todo](https://user-images.githubusercontent.com/93237922/221799724-c32983af-bce2-4212-88f6-87e9e33daa17.jpg)